packages/slice-machine/lib/models/common/widgets/StructuredText/index.ts

```diff
@@ -31,13 +31,21 @@ const Meta = {
   description: "A rich text field with formatting options",
 };
 
+const ManualFields = {
+  labels: { yupType: "array", validate: {} },
+};
+
 const schema = yup.object().shape({
   type: yup
     .string()
     .matches(/^StructuredText$/, { excludeEmptyString: true })
     .required(),
-  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-argument
-  config: createValidationSchema(removeProp(FormFields, "id")),
+  config: createValidationSchema(
```

Comment:
Why do we need it in the schema ?

I mean, from my understanding, this YUP is used to validate the forms in the save modal. We don't have a fields for label and data are merged later (after validation I believe).

Reply:
<img width="582" alt="image" src="https://user-images.githubusercontent.com/69144671/178452419-51693c95-7a49-4b3c-88d1-05d0e5b232f1.png">
it it's not in the yup schema it will fail on the `.schema.validateSync` method

Reply:
hmmm ok so it seems to validate the initialValue too, weird haha